plugins/goodreads.koplugin/doublekeyvaluepage.lua

```diff
@@ -231,24 +283,31 @@ function DoubleKeyValuePage:init()
         kv_page = self,
     }
     -- setup main content
-    self.item_margin = self.item_height / 4
+    self.item_margin = self.item_height / 6
     local line_height = self.item_height + 2 * self.item_margin
-    local content_height = self.dimen.h - self.title_bar:getSize().h
+    local content_height = self.dimen.h - self.title_bar:getSize().h - self.page_info:getSize().h
     self.max_loaded_pages = 1
     self.items_per_page = math.floor(content_height / line_height)
     self.pages = math.ceil(self.total_res / self.items_per_page)
     self.main_content = VerticalGroup:new{}
     self:_populateItems()
+
+    local content = OverlapGroup:new{
+        dimen = self.dimen:copy(),
+        VerticalGroup:new{
+            align = "left",
+            self.title_bar,
+            self.main_content,
+        },
+        footer,
+    }
     -- assemble page
-        self[1] = FrameContainer:new{
+    self[1] = FrameContainer:new{
         height = self.dimen.h,
         padding = padding,
         bordersize = 0,
         background = Blitbuffer.COLOR_WHITE,
-        VerticalGroup:new{
-            self.title_bar,
-            self.main_content,
-        },
+        content
```

Comment:
Might as well add a comma to possibly reduce the amount of lines changed in the future.